lib/read.js

```diff
@@ -170,6 +176,18 @@ function contentstream (req, debug, inflate) {
       stream = req
       stream.length = length
       break
+    case 'br':
+      if (hasBrotliSupport) {
+        stream = zlib.createBrotliDecompress()
+        debug('brotli decompress body')
+        req.pipe(stream)
+      } else {
+        throw createError(415, 'unsupported content encoding "' + encoding + '"', {
```

Comment:
It has been bugging me how to DRY this part up and I think I figured it out: the `stream` variable starts off `undefined` and is only set within these `case` statements. We could probably drop the `default` and then put it in a `if (!stream) { ... throw }` between the end of the `switch` and the `return`.

Reply:
Done